engine/menu/options.asm

```diff
@@ -404,7 +404,7 @@ Func_41ee9:
 	ld [hl], "▶"
 	ret
 
-Func_41f06:
+InitOptions:
```

Comment:
This duplicates a label at https://github.com/pret/pokeyellow/blob/master/engine/menu/main_menu.asm#L125
Sorry for taking over a year to look at this.

Reply:
Ah, thanks for taking a look at it dannye.  Do you have an alternative suggestion?

Reply:
The preexisting `InitOptions` is for initializing the player's option.
This routine appears to initialize the display of the options menu.
So perhaps `InitOptionsMenu`.
Be sure to run `make compare` to ensure the build passes.